app/graph/mutations/media_mutations.rb

```diff
@@ -4,8 +4,7 @@ module MediaMutations
     account_id: 'int',
     project_id: 'int',
```

Comment:
actually, we may no longer need to expose a `Media` mutation, since all handling will happen though `ProjectMedia`.